.travis.yml

```diff
@@ -32,7 +32,7 @@ matrix:
         - MATRIX_EVAL="CC=clang && CXX=clang++"
 
     - os: osx
-      osx_image: xcode6.4
```

Comment:
This must not be changed, we test on the lowest version available (we support macOS 10.9 w/ XCode 6.2)

Reply:
Two things about that change:
1) Travis has deprecated it.
2) It has an invalid SSL configuration that prevents it from downloading 'get-pip.py'

Reply:
```
Running builds with Xcode 6.4 in Travis CI is deprecated and will be removed in January 2019.
If Xcode 6.4 is critical to your builds, please contact our support team at support@travis-ci.com to discuss options.
```

```
Connecting to bootstrap.pypa.io|151.101.192.175|:443... connected.
ERROR: cannot verify bootstrap.pypa.io's certificate, issued by 'CN=GlobalSign CloudSSL CA - SHA256 - G3,O=GlobalSign nv-sa,C=BE':
  Issued certificate has expired.
To connect to bootstrap.pypa.io insecurely, use `--no-check-certificate'.
```